tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/ManualInstrumentation/Configuration/TracerSettings/AgentUriGetIntegration.cs

```diff
@@ -0,0 +1,36 @@
+﻿// <copyright file="AgentUriGetIntegration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System.ComponentModel;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.Telemetry;
+using Datadog.Trace.Telemetry.Metrics;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.ManualInstrumentation.Configuration.TracerSettings;
+
+/// <summary>
+/// System.Uri Datadog.Trace.Configuration.TracerSettings::get_AgentUri() calltarget instrumentation
+/// </summary>
+[InstrumentMethod(
+    AssemblyName = "Datadog.Trace.Manual",
+    TypeName = "Datadog.Trace.Configuration.TracerSettings",
+    MethodName = "get_AgentUri",
+    ReturnTypeName = "System.Uri",
+    ParameterTypeNames = new string[0],
```

Comment:
So far most have been `ParameterTypeNames = []` but this one is `new string[0]` I don't think there is a difference between the two though correct as both mean the same thing

Reply:
Yeah, they're the same, it depends on _when_ I generated this, because I updated the generator part way through 😃

Reply:
Ahhh makes sense thanks!

Reply:
Updated them for consistency, thanks!